### ISSUE TITLE
Fix false positives for longhand grid-column-* properties in value-keyword-case

### ISSUE THREAD

**Issue by lensbart** [stylelint/stylelint#4605]
Hello!

`value-keyword-case` flags the following instance:

```
grid-column-end: wideBlockRow;
```

However, the rule seems to be primarily designed to impose consistent use of CSS keywords.

Because camel casing seems desirable for (or at least improves the readability of) `longUserDefinedVariables`, it would be neat to disable this feature for such keywords.

Thank you!

**Comment by jeddy3** [comment]
@lensbart Thanks for the report.

I can reproduce this using [the demo](https://stylelint.io/demo#N4Igxg9gJgpiBcICGACYKDmAnAllAtJADYCuAtgHb4wVTwoDueMAQkRGANYBKEDA3PxQBfADoUQAGhABnAJ4UALkgAeCcDJlTwECgDMcGdcHEoUokFhJEYMi-RMUzZi0kX4rN-BQj4SFTh8GCnsURSsYSVNnCwAjdi5vXxgyAAdFOVDwkkjolx12LCT8HAoANyQiPHwACxgVLIiopxidMjIadx9qNIzGnObncxBYMCIkLDccXXx4jk5iqBJUqrA3GHxUrAhUmCxFHFtQgG0852zclqHHIaGLQx8sGBOz2+HIChkYMBIDso2liscGtFLZ8ExFDV8FAcHo9HtOvgKqQjiBXs4ALrosRXFAYwatUbjSYHGZzRLdGQ1CD7GpIWibba7fZyfAQf5YXCwOwIMJNM4WPS6dx6JBkHBEVndQGrdbeMWo+gXAn5IVKfCi8WS4rizSlDD4DA0PbAjViiWszgwOQMGlQfqXO4gPT+MCkqhrIhgYqlZF4B0q4Yuihu6Yeyre7r+GSpJBgGAEHZ7Nw0gMC52u934KoUGATQ2TGGI7o+T7KWgTAgwp6h3Rp3EWK1yPSTDrQ77EqYzbo4NI08uKetOjowpAavOKEhPeVtqMBIIhXnK9PS2zx2j6-Ax76w4E4PpL-kNkDS5ay0H4NwlPv7DzWRV8gYr3wy4Fyr42N00nlKo9O7opOkrIyBAU7xkOrQASo4Rjl84rELoP6Po6kG+L6lTVFAoHxBsMjjFShAQO0nRIcux5bEmLLFP4gR8Iuv5PseH7fIoNKbF8JBYYQ+EyNR850RB+TMV+RSpBxXEwDYHTqnOtHBIJwzCaxRQZLsfFyfRyGBhYMjhJuJYwAwObPIejFOv4+7qQuCmuO4ngbIBGTZqUGyxDAQpPC8uL5JUDBIHIPKBmYNxvMM9TxukXmha05KcDYmiXnooJFDICozq5CRxbYgXok6BhYLp3i2KC9pot5mJBa0Dw0iZ9CnOVTpIEleyEcRSgWLleLYmc+Lple9npYQSBfKEFjsAwewWNpyB2feg0xnGGxNclo3IEQfkBVu+peMZU19bNXhwcCEDsFQuZGS5iUrbyrjrf5OXHrF3EQDIm6xJM8Y9EBzm5rM7k1atuYcntj2Zc9r0UAa72LUVF2-ctk03WtG0PU6T1jC9b0fRs53GX9Hm1cMvn3fgZDWAcP0mdN6PsBDUPY1usafW5BOrcTm10ztLkg2jYNJhQWMw7jl0I1gbN3ZtZNEBTu1SOmT384Ln0LZ9oviyjW2Q1zuY8zFfO7ALkOzAzKuuf9nlI+zqOtAhRR1CoQ0jUj42I9Ntu1PU2Y0BgkKrVS-a6-kkBte4jmsnjLMA7y9VvLdKOB84IWxyA4UwJF0edaqOAFV0xUJh1DV4pV+TVRbdWZ4pGQ2Dm7jB2Q9JQDynVYuVOJDL1x514iDA1PuthMxspSvbA6v3QnFg-LpRGMpRTlh5T+NR+X5Vx2PcvlUntwWKn6fL9FPnNUUk+sWQM-MgexetPlhW5rp+dldFHfRfcGCPITMf764h+tdJg7r-vwwh7ME1hgbWGV5gF0ft1XET9CQdgmF2KgsR6QGlNldRG9ALBA1dumIkCCszIKNmgyOZciYS2tvkPBJIwyzDBkdBCZ1DJ4zVpbchpNyY4EpuPEY8DqFkjoSkY6p1GYwxYZg5GJNOYbFltNKhiDaHzC3IIhhIjmbm0JlgmAwN-5wLGPgmhT0pEL3rg7OR7okIAEZZG8PkU9GCEpNz0JOnWVh8cdGUJsVmFRwt4aH1HpLDhXD3HDDMTQlRaCxFkI1kYmRuDPFhOcVQYh6jAZaJwceUJMx54RxSRnFeEiAoJ2Cp1beKgIp-z3p-ZA39MmLkvlnHORU76lWbpfF+b8opVNFj-ToRSt4gCAbAEBYCFFcEgaFFubw24VXTMGWsHoiL11hswvxriSZSxltzYJgpMwJPaLBAe6CxZrI5ttaRWzppzK8Ysg5MMSEaJPGk45lzdkzBMV9JyxkkIAAYXkhizBQNKawnbiJds82ZryqCxieEoOoXxeI+MHp8Zg-jeIbM4bE48VyaHQs6HCsEaDBkPOwSlM5QS-nzPBL3UEETVniKttw0osAlCINCAAJmmjYSGkJ8AAC89i+DnPuGyIB3nZJcpY6aI4cBjnhG4KcGxwmHMiavQp2zRUJhleOeV04lV3NyeIkl3DpWyonAqwawKHlguNZq01OqNi4thbYAlhyiWpO0VK212rJzTkmJDDYlEUwpWVXSqJa9PWjm9eav1Ro2TMiDaos2rMTkUOGCa-AABHHIWBw44EKnXMciKjmovYdLDFFz0zpqzXsXN+abmJuLSm4Z5ydbqqrdm2ttd63JOTYap53CAInSpX3U2Irui6jptQBkEA9BblAlgT6iKx3kDckUGwSAYRG35dsVFA6V0tW6PYnMBpt0vRFRRc+rJLWrWteqga4rfr3M6Y1Nh6LyXok3k6HeFSUAf2fs6bON884tMLrA5Opd34Vy-slHp7UH4TOge3aaSk2JuD0rEX4rkPpWkULxQlyKR5IyNeqlDRQ0O4AwxeQNykG0qseR69MpHLyKHQ5huNyYaM9qXsMYjyHJIsTYsHWIpQE20uuvS8h3CmNCZE5xw5T6U1Sf4yJdsMh1xQHpF2sgwnAU0ZLDMUdpkUJCWUzRqodb9nLJFqG1VqadKmbYuZrTSyuOkM0QxpiDmihit6Lm2+oRfmMa8+xGAnFfB6ISte52fB0lOiY2JULXEIu8UdZCZ1eHXUEeJf2kjwWEtheoFJREUXQUxfBZ5z8NH8sSSK+qbxEBlDulWlhEgOElOVbYqpRVw0rVle4RZWuPWb19fVciHI+Amy2iwAQErwxb3TTG+cizSyi10atqWzZraFuVHG05n+tzVY2YKXhslmKnSLecstg7SbuPudi60C7e2fPfS+QF6IOJhAgGEEAA).

It seems we haven't accounted for the longhand `grid-column-end` and `grid-column-start`. The shorthand `grid-column` doesn't trigger the bug.

I believe the fix will only involve adding `grid-column-end` and `grid-column-start` to [this conditional](https://github.com/stylelint/stylelint/blob/master/lib/rules/value-keyword-case/index.js#L131) in the rule.

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

---

In the future, please use the [report a bug issue template](https://github.com/stylelint/stylelint/issues/new?template=REPORT_A_BUG.md).
